<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*horizontal ：水平QSlider*/
QSlider::groove:horizontal {
    border: 0px solid #bbb;
}

/*1.滑动过的槽设计参数*/
QSlider::sub-page:horizontal {
     /*槽颜色*/
    background: rgb(70,130,180);
     /*外环区域倒圆角度*/
    border-radius: 2px;
     /*上遮住区域高度*/
    margin-top:8px;
     /*下遮住区域高度*/
    margin-bottom:8px;
}

/*2.未滑动过的槽设计参数*/
QSlider::add-page:horizontal {
    /*槽颜色*/
    background: rgb(105,105, 105);
   
    border: 1px solid #777;
    /*外环区域倒圆角度*/
    border-radius: 2px;
     /*上遮住区域高度*/
    margin-top:8px;
     /*下遮住区域高度*/
    margin-bottom:8px;
}

/*3.平时滑动的滑块设计参数*/
QSlider::handle:horizontal {
    /*滑块颜色*/
    background: rgb(193,204,208);
    /*滑块的宽度*/
    width:20px;

    /*滑块外环为1px，再加颜色*/
    border: 1px solid rgb(193,204,208);

     /*滑块外环倒圆角度*/
    border-radius: 55px;
     /*上遮住区域高度*/
    margin-top:2px;
     /*下遮住区域高度*/
    margin-bottom:2px;
}

/*4.手动拉动时显示的滑块设计参数*/
QSlider::handle:horizontal:hover {
    /*滑块颜色*/
    background: rgb(170, 255, 0)
    /*滑块的宽度*/
    width: 20px;
    
    border: 1px solid #484848;
     /*滑块外环倒圆角度*/
    border-radius: 15px;
     /*上遮住区域高度*/
    margin-top:2px;
     /*下遮住区域高度*/
    margin-bottom:2px;
}

</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>440</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>13</pointsize>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>00 : 00 / 00 : 00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>500</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QListWidget" name="music_list">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open_folder_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件夹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_file_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QSlider" name="volume_adjust_slider">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>390</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="play_mode_select">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>500</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
